torch/csrc/jit/passes/fold_linear_bn.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <torch/csrc/jit/api/module.h>
+
+namespace torch {
+namespace jit {
+
+struct TORCH_API LinearBNParameters {
```

Comment:
this is a super minor nit - but from what I can tell the contents of fold_linear_bn.(h|cpp) are essentially implementation details of frozen_linear_folding.cpp, is that correct? if so we can probably move this into the anonymous namespace of frozen_linear_folding.cpp right?

(this is mostly just a matter of preference tbh)

Reply:
That's correct. Actually, `fold_linear_bn.h` is mostly copied from `fold_conv_bn.h`, so I thought I would follow it's style. I'm fine with either options -- what do you think?

Reply:
I guess it's fine as it is